agignore

```diff
@@ -1,3 +1,4 @@
 log
 tags
 tmp
+/.git/
```

Comment:
I'm in favor of removing the leading slash - git repos can be nested with submodules, or I might want to run ctrl-p from a folder that contains multiple repos.